README.md

```diff
@@ -28,7 +28,28 @@ $ npm i egg-utils
 
 ## API
 
-### `#getFrameworkOrEggPath(cwd[, eggNames])`
+### `getPlugins(options)`
```

Comment:
could support an option to get all plugins including disabled

Reply:
it just returned all plugins.